.travis.yml

```diff
@@ -19,8 +19,7 @@ services:
 #   - docker_images
 
 before_install:
-- composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts
-- sudo service mysql stop
```

Comment:
I originally added this because I had port conflicts, I am surprised it didn't break the build 🤔

Reply:
Any idea what was I missing?

Reply:
I am not sure but I expect to make appwrite container connect to the database through the docker network itself and not through the host network. Where I can find the database configuration related to the test case to check the host URL?

Reply:
Sorry I missed a point, the test cases will use the same database communication as defined in the application config itself which should be within the docker network that's why there is no need to check or disable the local mysql instance